From eb3bcf3bcd791c1a9a8732dfb41514f33a1446d0 Mon Sep 17 00:00:00 2001
From: Nir Soffer <nirsof@gmail.com>
Date: Tue, 26 Jun 2018 01:43:01 +0300
Subject: [PATCH] v2v: rhv-upload-plugin: Remove unused variables

Remove some instances of "transfer" and "transfer_service" that are not
used in current code.

(cherry picked from commit 32ab53cecd78ee5c140f17e5792710698627d265)
---
 v2v/rhv-upload-plugin.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/v2v/rhv-upload-plugin.py b/v2v/rhv-upload-plugin.py
index 10887c031..5c036c46a 100644
--- a/v2v/rhv-upload-plugin.py
+++ b/v2v/rhv-upload-plugin.py
@@ -263,7 +263,6 @@ def request_failed(h, r, msg):
 def pread(h, count, offset):
     http = h['http']
     transfer = h['transfer']
-    transfer_service = h['transfer_service']
 
     headers = {"Range", "bytes=%d-%d" % (offset, offset+count-1)}
     # Authorization is only needed for old imageio.
@@ -309,7 +308,6 @@ def pwrite(h, buf, offset):
 
 def zero(h, count, offset, may_trim):
     http = h['http']
-    transfer = h['transfer']
 
     # Unlike the trim and flush calls, there is no 'can_zero' method
     # so nbdkit could call this even if the server doesn't support
@@ -371,7 +369,6 @@ def emulate_zero(h, count, offset):
 
 def trim(h, count, offset):
     http = h['http']
-    transfer = h['transfer']
 
     # Construct the JSON request for trimming.
     buf = json.dumps({'op': "trim",
@@ -394,7 +391,6 @@ def trim(h, count, offset):
 
 def flush(h):
     http = h['http']
-    transfer = h['transfer']
 
     # Construct the JSON request for flushing.
     buf = json.dumps({'op': "flush"}).encode()
-- 
2.17.2

